src/providers/vod/vod-provider.ts

```diff
@@ -16,9 +16,14 @@ import {CaptionLoader} from './caption-loader';
 export class VodProvider extends Provider {
   private _fetchedCaptionKeys: Array<string> = [];
   private _fetchingCaptionKey: string | null = null;
+  private _cuePointsData: any[] = [];
+  private _lastTimeUpdated: number = 0;
+  private _isPreventSeekActive: boolean;
 
   constructor(player: Player, eventManager: EventManager, logger: Logger, types: CuepointTypeMap) {
     super(player, eventManager, logger, types);
+    // @ts-ignore
+    this._isPreventSeekActive = player.preventSeekOptions.isActive;
```

Comment:
whats the error ?

Reply:
it is dependent on a new API in kaltura-player- `preventSeekOptions`.
since it does not exist yet I had to add ts-ignore, but when I will first merge kaltura-player PR, i will then remove the annotation. same in timeline PR..